pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -69026,7 +69026,6 @@ self: {
        description = "Non-GC'd, contiguous storage for immutable data structures";
        license = lib.licenses.bsd3;
        hydraPlatforms = lib.platforms.none;
-       broken = true;
```

Comment:
So that this change does not get overwritten, please remove `compact` from `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml` and run `maintainers/scripts/haskell/regenerate-transitive-broken-packages.sh`.

Reply:
Sorry I missed this, should be all set now.